### ISSUE TITLE
Is the term "a11y" a barrier of inclusion and understanding?

### ISSUE THREAD

**Issue by svinkle** [a11yproject/a11yproject.com#504]
## Issue

This issue is regarding this thread on Twitter. Please have a read through first (multiple links since I still don't understand Twitter threading.)

https://twitter.com/jacobwarduk/status/896714540855046144
https://twitter.com/A11YProject/status/896793288858112000
https://twitter.com/stefsull/status/896811103363112960

### Tl;dr:
Seeing "a11y" on the web and elsewhere is potentially confusing and people may not understand what it really means, causing a barrier of inclusion.

## Porposal
I'd like to propose a discussion on seeing how others feel about this: if it's indeed an issue and whether we should consider taking action to be more inclusive in our branding and content.

For example:
```html
<abbr title="accessibility">a11y</abbr>
```

Or, do we just write "accessibility" instead of "a11y" where we can?

### Note:
Everyone and anyone reading this is welcome to chime in on this conversation. I believe this could be an issue with the greater accessibility community as a whole.

**Comment by patrickhlauke** [comment]
any acronym is a barrier. TDD, PWA, etc. but even the full term can be a barrier. developers aren't ignorant of accessibility because they don't know the acronym a11y. they're usually just as ignorant about the term "accessibility" itself. ask a non-expert what they think "accessibility" is, and they're likely going to draw a blank or start talking about server uptime or how a site can be accessed on mobile as well as on desktop or something.

in short, i think the use of the acronym is not the crucial part of why people don't flock to the topic/site.

**Comment by aardrian** [comment]
Patrick beat me to it. All abbreviations are barriers. Using ‘a11y’ outside of the industry is the bigger issue. When I give talks to general audiences (web devs, devs, UXers, etc) I open my slide decks by defining it. Thankfully I have noticed others are now doing the same (not meant to imply I kicked off the trend, just that perhaps as an industry we are finally recognizing this).

Now, all that being said, I am not in favor of any code solution like `<abbr title="accessibility">a11y</abbr>`. Instead, just use plain writing to define it on first use throughout all stand-alone pages, such as this fake quote:

> This page about the thing has a feature for accessibility (a11y) that words are thinging for…

Maybe link "a11y" to a disambiguation / definition page.

**Comment by scottaohara** [comment]
completely agree with Patrick and Adrian here.

a11y is a tiny bump to get in the door, but it's in no way blocking anyone, unless they are unwilling to ask or search for what it means. 

As for the a11yproject specifically, maybe a basics article on numeronyms (a11y, i18n, etc)?  (doing a quick search, I found an article that points out [10 numeronyms devs should know](http://www.hongkiat.com/blog/tech-numeronyms/)). Maybe adding a bit more of an intro to the website that better calls out, like Adrian's example, of "accessibility (a11y)" to drive it home that they're related.  

That's all I got for now

**Comment by aardrian** [comment]
Ooh, to Scott's point about an explainer, I completely forgot that I wrote one a while ago after seeing mass confusion (maybe, like, two people) on Stack Overflow: http://adrianroselli.com/2016/11/a11y-accessibility.html

Just steal that page and recreate it here.

**Comment by ryanve** [comment]
I think `a11y` is curious enough that if someone doesn't know it they're likely to google it and remember it after that. So my vote would be to keep using a11y :)

**Comment by iandevlin** [comment]
As others have said, all acronyms can cause issues when people are unaware of what they mean, but I would also like to mention a question I asked on Twitter a while back as to how people pronounce "a11y".
For me, I would always say "accessibility", but some replied that they say "a eleven y" or "ally", which only adds to the confusion in my book.

**Comment by dpersing** [comment]
A short explanation on the About page and  would go a long way to clarifying when folks first arrive. I think the short text that appears on the homepage and in the meta description confirms the purpose of the project for users first discovering the site, however, even if it doesn't go into detail about the acronym.

**Comment by jacobwarduk** [comment]
@ryanve if somebody has to leave your website to Google the meaning of the primary term of your website, I think there's a problem with that.

**Comment by jacobwarduk** [comment]
As the starter of that 'thread on Twitter, I stand by what I said. And think that the term should at least be defined somewhere obvious, such that people coming to this understand what it means.

I do like @aardrian's suggestion of a disambiguation/definition of terms page.

**Comment by svinkle** [comment]
Thanks for all your input, everyone!

Closing via #517 and #518.
